lib/mindwendel_web/live/idea_live/index_component.ex

```diff
@@ -18,4 +19,15 @@ defmodule MindwendelWeb.IdeaLive.IndexComponent do
 
     {:noreply, socket}
   end
+
+  def handle_event("label", %{"id" => id, "label" => label}, socket) do
+    Brainstormings.get_idea!(id)
+    |> Brainstormings.update_idea(%{label: label})
```

Comment:
broadcast is missing in the update_idea function, see add_like function

Reply:
Thank you.

Reply:
Done